dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,155 @@
+[[dev-guide-event-notifications]]
+= Event Notifications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+For each of your projects, you can choose to receive email notifications about
+various xref:../dev_guide/events.adoc#dev-guide-events[events], including:
+
+* _Dead deployment_: A failed deployment in which no other deployment in the
+rollout history is able to remain active. The related `deploymentConfig` has no
+deployments with active replicas.
+
+* _Failed deployment_: A new deployment fails and another deployment in the
+ rollout history is now active. The related `deploymentConfig` has one active
+ deployment, but it is not the latest deployment.
+
+* _Dead build_: Occurs when automatically triggered builds are failing for a
+specified `buildConfig` that previously had successful builds.
```

Comment:
If we are moving to more readable resource names, this should probably be "build config"